From 4ad91b9e3464b46a3053bd5586d43292c5fabd67 Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Fri, 17 Jan 2020 10:04:38 +0530
Subject: [PATCH] Enable Smart Replies for all

The smart replies were disabled by dev citing
> these are intended to be made available only for smart watches, but on P they show up in the notification UI on the phone and a lot of people found that to be quite annoying

But I have fixes for it included in ROM framework
---
 .../java/com/moez/QKSMS/common/util/NotificationManagerImpl.kt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/presentation/src/main/java/com/moez/QKSMS/common/util/NotificationManagerImpl.kt b/presentation/src/main/java/com/moez/QKSMS/common/util/NotificationManagerImpl.kt
index f94eb3d3..bb6fd950 100644
--- a/presentation/src/main/java/com/moez/QKSMS/common/util/NotificationManagerImpl.kt
+++ b/presentation/src/main/java/com/moez/QKSMS/common/util/NotificationManagerImpl.kt
@@ -364,10 +364,7 @@ class NotificationManagerImpl @Inject constructor(
         val responseSet = context.resources.getStringArray(R.array.qk_responses)
         val remoteInput = RemoteInput.Builder("body")
                 .setLabel(title)
-
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
             remoteInput.setChoices(responseSet)
-        }
 
         return NotificationCompat.Action.Builder(R.drawable.ic_reply_white_24dp, title, replyPI)
                 .setSemanticAction(NotificationCompat.Action.SEMANTIC_ACTION_REPLY)
